test/fixtures/mson/enum-variants.json

```diff
@@ -334,9 +367,17 @@
                           "enumerations": {
                             "element": "array",
                             "content": [
+                              {
+                                "element": "string",
+                                "content": "S"
+                              },
                               {
                                 "element": "string",
                                 "content": "T"
+                              },
+                              {
+                                "element": "string",
+                                "content": "U"
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?